sw/host/spiflash/updater.cc

```diff
@@ -42,6 +42,22 @@ uint32_t Populate(uint32_t frame_number, uint32_t code_offset,
   return copy_size;
 }
 
+/**
+ * Copy the hash `src` into `dst` using a byte order consistent with the hash
+ * produced by the code signing tool.
+ */
+void CopyHash(const uint8_t *src, uint8_t *dst) {
+  const uint32_t *wp = reinterpret_cast<const uint32_t *>(src);
+  uint8_t *p = dst;
+  for (size_t i = 0; i < 8; ++i) {
+    uint32_t tmp = wp[7 - i];
+    *p++ = (uint8_t)(tmp >> 24);
+    *p++ = (uint8_t)(tmp >> 16);
+    *p++ = (uint8_t)(tmp >> 8);
+    *p++ = (uint8_t)(tmp >> 0);
+  }
+}
+
```

Comment:
similar to `cw_spiflash.py`, we just need to reverse the byte order. Would the suggestion below work?

Reply:
Updated to use `std::reverse`.